src/engraving/dom/spanner.cpp

```diff
@@ -65,6 +65,11 @@ SpannerSegment::SpannerSegment(const SpannerSegment& s)
     m_spannerSegmentType = s.m_spannerSegmentType;
     m_p2                 = s.m_p2;
     m_offset2            = s.m_offset2;
+
+    // Parent of SpannerSegment is a System. It will be
```

Comment:
Ideally, we should move this outside of the constructor and do it in a slightly different way:

```
SpannerSegment* copy = segment->clone();
copy->setParent(score->dummy()->system());
```

Reply:
done 👍